src/OpenTelemetry.Exporter.Jaeger/OpenTelemetry.Exporter.Jaeger.csproj

```diff
@@ -13,13 +13,19 @@
     </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="ApacheThrift" Version="0.13.0.1" PrivateAssets="All" />
+    <PackageReference Include="ApacheThrift" Version="0.13.0.1" ExcludeAssets="all" GeneratePathProperty="true" />

```

Comment:
is the change in this file related to the overall PR?

Reply:
I would say... half related :) The sample/test project added on this PR is using JaegerExporter. It blows up with an assembly load exception because the ApacheThrift library doesn't flow to parent. Bug introduced on #494 when I removed the fork. I added "PrivateAssets" at the time because it was there originally (https://github.com/open-telemetry/opentelemetry-dotnet/pull/494/files#diff-4eb94e2c56833f774f865883d81cc1d9L17) but I think there's a subtle difference between ProjectReference and PackageReference in how dependencies flow.

Reply:
my point was that we needed this change even without asp.net collector and maybe sooner. If this PR will go fast, it's OK to merge together

Reply:
Opened a dedicated PR for the fix.